docs/index.rst

```diff
@@ -1444,12 +1444,16 @@ This library provides a *Repository Proxy* that wraps your object repositories t
 Assertions
 ~~~~~~~~~~
 
-Your object factory's have helpful PHPUnit assertions:
+Both object proxies and your ModelFactory have helpful PHPUnit assertions:
 
 .. code-block:: php
 
     use App\Factory\PostFactory;
 
+    $post = PostFactory::createOne();
+    $post->assertPersisted();
+    $post->assertNotPersisted();
```

Comment:
```suggestion
    $post->_assertPersisted();
    $post->_assertNotPersisted();
```

We need to show the new method names, correct?

Reply:
yes of course